src/sql/workbench/contrib/query/browser/queryActions.ts

```diff
@@ -700,6 +700,9 @@ export class ListDatabasesActionItem extends Disposable implements IActionViewIt
 			return;
 		}
 
+		// Remove the pool instance from database name in the dropdown
```

Comment:
the dbName should only contain the actual database name, we shouldn't be handling this here.

Reply:
this is the actual database name

Reply:
Then why does the dropdown only contain the first part of the name? And why is this logic + the logic that Karl added necessary?

Reply:
the first part of the name is the database name. The part after "@" is the pool name. Any database can be connected with any pool. We will always want to connect to the database via the pool of the current connection, and that's why we remove the pool. If the user wants to use another pool, they would have to use the connection dialog.